docs/README.rst

```diff
@@ -0,0 +1,59 @@
+:orphan:
+
+======================================
```

Comment:
This will generate a warning (number of `=`s).

Reply:
There are other occurrences of this "warning".

Reply:
Issues like that can be easier to detect if CI is set up to build docs; see https://github.com/scrapy/scrapy/blob/b6eae22b38e2f2bec248c241342f6f9f583c9833/tox.ini#L73

Reply:
Updated here https://github.com/ejulio/itemloaders/pull/5/commits/c8357ee23041ff0ea49051dc151d7429c5793541